.github/workflows/pr-ci.yml

```diff
@@ -1,32 +1,44 @@
 name: CI build and push (PR)
 
 concurrency:
-  group: pr-${{ github.event.number }}
+  group: ci-${{ github.run_id }}
   cancel-in-progress: true
 
 on:
```

Comment:
We also want to retain the current behaviour of `safe-to-test`.
```yaml
  pull_request_target:
    types:
      - opened
      - reopened
      - synchronize
      - labeled
      - unlabeled
    branches:
      - main
      - v[0-9]+
      - v[0-9]+.[0-9]+
      - cryostat-v[0-9]+.[0-9]+
```

Reply:
Also, there might be issues accessing `${{ secrets.GHCR_PR_TOKEN }}` on `issue_comment` event. See https://github.com/cryostatio/cryostat/pull/1599/files#r1282452369

Reply:
I thought we only wanted to build and run tests when there's a comment with the /build_test command and not when there's other PR events?

Reply:
I thought we only wanted to build and run tests when there's a comment with the /build_test command and not when there's other PR events?